lib/ui/dart_ui.gni

```diff
@@ -12,6 +12,7 @@ dart_ui_files = [
   "//flutter/lib/ui/isolate_name_server.dart",
   "//flutter/lib/ui/key.dart",
   "//flutter/lib/ui/lerp.dart",
+  "//flutter/lib/ui/math.dart",
```

Comment:
This file was added in https://github.com/flutter/engine/pull/36106, but I forgot to add it here.  To be honest, I have no idea why it didn't blow up the framework build, but it didn't. Anyhow, adding it in now so that we can finish moving `clampDouble` into `dart:ui`.